setup.py

```diff
@@ -11,8 +11,11 @@ def get_requirements(file):
     with open(file) as f:
         for l in f.readlines():
             l = l.strip()
-            if l and not l.startswith("#"):
-                res.append(l)
+            if l:
+                if l.startswith("-r"):
+                    res.extend(get_requirements(l[3:].strip()))
```

Comment:
Its common in python to write `pip install -rrequirements.txt` In this case, the proper index would be 2 instead of 3.